cocos/renderer/CCGLProgramState.h

```diff
@@ -241,6 +241,9 @@ class CC_DLL GLProgramState : public Ref
 {
     friend class GLProgramStateCache;
 public:
+    /** x-studio365 spec, update node program state */
+    static GLProgramState* getPositionTextureColorGLProgramState(Texture2D* texture, bool noMVP = true);
+    static GLProgramState* getPositionTextureGrayGLProgramState(Texture2D* texture);
```

Comment:
It's just wrapper for easy to update any other subclass of Node's position_texture_color shaders, do you have a good idea for do this?